jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml

```diff
@@ -5,14 +5,16 @@ metadata:
     opendatahub.io/notebook-image: "true"
   annotations:
     opendatahub.io/notebook-image-url: "https://github.com/thoth-station/s2i-generic-data-science-notebook"
-    opendatahub.io/notebook-image-name: "Generic Data Scient Notebook Image"
+    opendatahub.io/notebook-image-name: "Standard Data Science"
     opendatahub.io/notebook-image-desc: "Jupyter notebook image with a set of data science libraries that advanced AI/ML notebooks will use as a base image to provide a standard for libraries avialable in all notebooks"
   name: s2i-generic-data-science-notebook
 spec:
   lookupPolicy:
     local: true
   tags:
   - annotations:
+      opendatahub.io/notebook-software: [{"name":"Python","version":"v3.8.3"}]
```

Comment:
I'm getting the error below when applying these annotations.  It looks like annotations values values can only be strings so these need to be single quoted in order to apply correctly